puppet-client.html

```diff
@@ -239,7 +232,8 @@
             }
         },
         onLocalChange: function(patches) {
-            this.propagateDomBind(patches);
+            // todo #26
+            this.propagateDomBind({patches: patches, results: patches.map(function() { return {};})});
```

Comment:
That's one of many reasons I do not like the fact the result is a map (https://github.com/Starcounter-Jack/JSON-Patch/pull/110#issuecomment-227387138)

What is more that is information ppc passes to ppc, no Polymer at this point. What's more ppc in here pretends it passed jsonpatch-API-compliant method at patches, what didn't take place!
Even though we are working with replaces only, we can still remove something.